pylibui/libui/box.py

```diff
@@ -54,13 +54,13 @@ def uiBoxDelete(box, index):
 # - int uiBoxPadded(uiBox *b);
 def uiBoxPadded(box):
     """
-    Sets the padding of the box.
+    Returns whether the box is padded.
 
     :param box: uiBox
     :return: int
     """
 
-    return clibui.uiBoxPadded(box)
+    return bool(clibui.uiBoxPadded(box))
```

Comment:
Definitely, let's keep everything inside pylibui/libui to be just very very thin wrappers around the C lib.

In fact, the most important thing to be accomplished is to finish the wrappers completely such that they can then be implemented with a more pythonic interface in pylibui/controls/, as we have been doing.